prow/github/types.go

```diff
@@ -68,6 +68,22 @@ const (
 	MergeSquash PullRequestMergeType = "squash"
 )
 
+func unmarshalClientError(b []byte) error {
+	var err error
+	clientError := ClientError{}
+	err = json.Unmarshal(b, &clientError)
+	if err == nil {
+		return clientError
+	}
+	alternativeClientError := AlternativeClientError{}
+	err = json.Unmarshal(b, &alternativeClientError)
+	if err == nil {
+		return alternativeClientError
+	}
+	// first Unmarshal() error is lost, should be similar with other struct
```

Comment:
We don't have to, right? This is the sort of rare thing that is going to be super frustrating to debug without proper logs. Lets just concat the two errors if we get both and emit them together